### ISSUE TITLE
Prazna stran pri tiskanju

### ISSUE THREAD

**Issue by matijapretnar** [ul-fmf/urnik#99]
Če greš na nek urnik in izbereš _Natisni stran_
<img width="1358" alt="Screenshot 2020-10-01 at 06 40 23" src="https://user-images.githubusercontent.com/1502128/94768901-056c2400-03b1-11eb-8b7c-3d592c2823ba.png">
dobiš prazno stran, na kateri je samo naslov
<img width="1377" alt="Screenshot 2020-10-01 at 06 41 32" src="https://user-images.githubusercontent.com/1502128/94768955-2af92d80-03b1-11eb-822f-89a03678894d.png">

**Comment by jureslak** [comment]
Ja, se strinjam. Bom pogledal kaj je narobe, zna biti da je tedenski pogled kaj pokvaril...

**Comment by jureslak** [comment]
Trenuten workaround: meni dela v Google Chrome, v Firefoxu ne, predvidevam, da si ti uporabljal Safari?

**Comment by jO-Osko** [comment]
Meni na Macu v Google Chrome tudi ne deluje.

**Comment by matijapretnar** [comment]
Tole se pa zgodi na Firefoxu (sklepam iz informacije `Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/85.0.4183.121 Safari/537.36` v PDFju):
[temen-urnik.pdf](https://github.com/ul-fmf/urnik/files/5326883/temen-urnik.pdf)

**Comment by matijapretnar** [comment]
Popravek, glede na https://developers.whatismybrowser.com/useragents/parse/#parse-useragent gre za Chrome

**Comment by jureslak** [comment]
Hmm, hiter workaround je, da izklopiš tiskanje ozadij.
